packages/nimble-components/src/tabs/tests/tabs.spec.ts

```diff
@@ -8,4 +33,131 @@ describe('Tabs', () => {
     it('can construct an element instance', () => {
         expect(document.createElement('nimble-tabs')).toBeInstanceOf(Tabs);
     });
+
+    it('setting activeid should scroll the active tab into view', async () => {
+        const { element, connect, disconnect } = await setup();
+        await connect();
+        const tabsPageObject = new TabsPageObject(element);
+        await tabsPageObject.setTabsWidth(300);
+        await waitForUpdatesAsync(); // wait for the resize observer to fire
+        element.activeid = '6'; // scrolls to the last tab
+        await waitForUpdatesAsync();
+        expect(
+            element.shadowRoot!.querySelector('.tablist')!.scrollLeft
+        ).toBeGreaterThan(0);
+
+        await disconnect();
+    });
+
+    describe('Scroll buttons', () => {
+        let tabsPageObject: TabsPageObject;
+        let element: Tabs;
+        let connect: () => Promise<void>;
+        let disconnect: () => Promise<void>;
+
+        beforeEach(async () => {
+            ({ element, connect, disconnect } = await setup());
+            await connect();
+            tabsPageObject = new TabsPageObject(element);
+        });
+
+        afterEach(async () => {
+            await disconnect();
+        });
+
+        it('should not show scroll buttons when the tabs fit within the container', () => {
+            expect(tabsPageObject.areScrollButtonsVisible()).toBe(false);
```

Comment:
nit: Up to you, but I think the reporting is _marginally_ nicer when you use `toBeFalse()` and `toBeTrue()`.